### PR TITLE
Add Chessboard controller and view

### PR DESCRIPTION
**Pull request by joewhale**
# Created Chessboards view (with divs) and Chessboards Controller

### CODE REVIEW

File: app/views/chess_boards/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+
+  </head>
+  <body>
+    <br><br><br>
+
+
+    <div class="chessboard">
+<!-- 1st -->
+<div class="white white-top white-left"></div>
```
Comment (lasley) [inline app/views/chess_boards/index.html.erb]: Lots of repeated code here. General rule of thumb - if you are ever copy/pasting, you are doing it wrong.

In this instance, you can loop 0-8, with an inner 0-8 loop. You can then identify black/white by using modulo. Hint on the modulos is `4 % 2 == 0 && 5 % 2 == 1`

**Comment by GonSanchezS** [comment]
I think we need to be super clear how are we going to write CSS. A few guidelines I, personally, like to follow: 

- Use [BEM](http://getbem.com/introduction/) (or bloc, element, modifier) when naming elements. A quick guide [here](https://css-tricks.com/bem-101/).
- Have a master file where we specify colors, column structure, font families (in this case, Roboto and Roboto Slug), so we can then apply them throughout the document using SASS variables. 
- Separate CSS files by section – for instance, one for header, one for footer, one for landing, one for each controller, etc. 

Maybe a topic to discuss during our next meeting.

**Comment by lasley** [comment]
The master file for colors and stuff is typically called `variables.less` or `variables.sass`, whichever you decide to use

**Comment by kellylougheed** [comment]
@joewhale would it also be possible for you to post a screenshot of what the board looks like on your local machine?

**Comment by joewhale** [comment]
screen shot 2016-11-22 at 9 39 43 am

File: db/schema.rb [hunk 2]
```diff
@@ -11,14 +11,11 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-
-
 ActiveRecord::Schema.define(version: 20161115154942) do
-  
-  
+
```
Comment (lasley) [inline db/schema.rb]: It is best to not make frivolous stylistic changes such as this. An example of why can be seen in the merge conflict for schema, which would not exist had this file not been edited.

**Comment by PamBWillenz** [comment]
Are we going to make the chessboard available/accessible to the games#show page? My understanding is that the pieces will be displayed on the same page as the chessboard? Or will they have separate views and we render one from the other?

**Comment by joewhale** [comment]
I'm still trying to figure out how to fix the borders around the chessboard.

**Pull request merged**